sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -105,11 +108,112 @@ describe("[AAD] TextAnalyticsClient", function() {
         })
       );
       const allInputs = enInputs.concat(esInputs);
-
       const results = await client.analyzeSentiment(allInputs);
       assert.equal(results.length, testDataEn.length + testDataEs.length);
       assertAllSuccess(results);
     });
+
+    it("client gets positive mined opinions", async () => {
+      const documents = [
+        {
+          text: "It has a sleek premium aluminum design that makes it beautiful to look at.",
+          id: "0",
+          language: "en"
+        }
+      ];
+      const results: AnalyzeSentimentResultArray = await client.analyzeSentiment(documents, {
+        mineOpinions: true
+      });
+      assert.equal(results.length, 1);
+      assertAllSuccess(results);
+      const documentSentiment: AnalyzeSentimentSuccessResult = results[0] as AnalyzeSentimentSuccessResult;
+      documentSentiment.sentences.map((sentence) =>
+        sentence.minedOpinions?.map((opinion) => {
+          const aspect = opinion.aspect;
+          assert.equal("design", aspect.text);
+          assert.equal("positive", aspect.sentiment);
+          assert.isAtLeast(aspect.confidenceScores.positive, 0);
+          assert.isUndefined(aspect.confidenceScores.neutral);
+          assert.isAtLeast(aspect.confidenceScores.negative, 0);
```

Comment:
another check we could add is that all confidence score add up to 1

Reply:
done.